src/Meshes/transformNode.ts

```diff
@@ -405,6 +405,9 @@ export class TransformNode extends Node {
             absolutePositionZ = absolutePosition.z;
         }
         if (this.parent) {
+            if (this.parent.parent && this.parent instanceof TransformNode) {
+                this.parent.setAbsolutePosition(this.parent.getAbsolutePosition());

```

Comment:
not sure to get the point of that line :D

Reply:
I got rid of it, I was thinking the function should be run for each ancestor, which initially looked like it made things work, but I had forgotten that the issue is only when the explosion is set within the viewer.onModelLoadedObservable.add() function. Creating the explosion in console after everything is loaded, works.